src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -73,35 +72,25 @@ def __init__(
         self._container_id = container_id
         self._container_type = container_type
         self._backend = backend
+        self._lock = lock
```

Comment:
No need to store it (and `sleep_time` as well)

Reply:
Done.